colossus/src/main/resources/reference.conf

```diff
@@ -48,7 +48,7 @@ colossus{
       request-buffer-size : 100
       log-errors : true
       request-metrics : true
-      max-request-size : "1 MB"
+      max-request-size : "1000 MB"
```

Comment:
hmm, I'd like to hear other opinions on this.  Personally I think it's probably better to keep this small but make it clear how to increase it.  Not a big deal since I think you should always tune this for each situation, but if I had to choose between accidentally rejecting a bunch of requests because this was defaulted to too low vs crashing a server because it ran out of memory trying to accept lots of large requests, I'd choose the former.

1MB was probably too small though, maybe 10MB?